Priyanker/Assignment No 2/Project2.xml

```diff
@@ -0,0 +1,128 @@
+<?xml version="1.0" encoding="utf-8"?>
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent" >
+
+    <TextView
+        android:id="@+id/title"
+        android:layout_width="300dp"
+        android:layout_height="37dp"
+        android:text="@string/fcc"
+        android:textSize="30sp"
+        android:paddingLeft="50dp"
+        android:textAlignment="center"
+        android:textColor="@android:color/white"
+        android:background="#006203"
+        android:fontFamily="sans-serif"
+        android:layout_alignParentTop="true"/>
+    <ImageView
+        android:id="@+id/symbol"
+        android:layout_width="39.3dp"
+        android:layout_height="39.9dp"
+        android:layout_toRightOf="@id/title"
+        android:src="@drawable/fccsymbol"
+        android:layout_alignParentTop="true" />
+    <TextView
```

Comment:
This extra bit of TextView is not necessary. What you can do instead is add alignParentEnd and marginEnd to the ImageView above. By doing so your layout will be consistent on different screen sizes. For Example:
```
        android:layout_marginEnd="30dp"
        android:layout_alignParentEnd="true"
```